pkg/strategy/bollmaker/strategy.go

```diff
@@ -78,6 +78,20 @@ type Strategy struct {
 	// AskSpread overrides the spread setting, this spread will be used for the sell order
 	AskSpread fixedpoint.Value `json:"askSpread,omitempty"`
 
+	// DynamicSpreadWindow enables the automatic adjustment to bid and ask spread.
+	// When DynamicSpreadWindow is set and is larger than 0, the spreads are calculated based on the SMA of amplitude of
+	// [DynamicSpreadWindow] K-lines
+	DynamicSpreadWindow int `json:"dynamicSpreadWindow,omitempty"`
```

Comment:
then, to unmarshal the dynamic spread settings, you can define a new struct called `DynamicSpreadSettings`